olive/model/__init__.py

```diff
@@ -183,6 +172,14 @@ def validate_type(cls, v):
             raise ValueError(f"Unknown model type {v}")
         return v
 
+    def get_resource_keys(self):
+        cls = REGISTRY[self.type.lower()]
```

Comment:
Is this equivalent to just returning `self.resource_keys`? The class attribute should be accessible using `self.` also.

Reply:
Please ignore this. Did not see that this is a method of `ModelConfig` and not `OliveModel`.